helix-view/src/editor.rs

```diff
@@ -177,6 +177,9 @@ pub struct FilePickerConfig {
     pub deduplicate_links: bool,
     /// Enables reading ignore files from parent directories. Defaults to true.
     pub parents: bool,
+    /// Enables reading an ignore file from `.helix/ignore`.
+    /// Whether to hide files listed in .helix/ignore in file picker and global search results. Defaults to true.
+    pub helix_ignore: bool,
     /// Enables reading `.ignore` files.
```

Comment:
this doesn't need a config options. This file is helix specific, if you don't want to read it then delete it